suitcase/mongo_normalized/__init__.py

```diff
@@ -48,22 +53,28 @@ def __init__(self, metadatastore_db, asset_registry_db,
             assets_db = _get_database(asset_registry_db, tls)
         else:
             assets_db = asset_registry_db
-        self._run_start_collection = mds_db.get_collection('run_start')
-        self._run_start_collection_revisions = mds_db.get_collection('run_start_revisions')
-        self._run_stop_collection = mds_db.get_collection('run_stop')
-        self._event_descriptor_collection = mds_db.get_collection(
-                                                        'event_descriptor')
-        self._event_collection = mds_db.get_collection('event')
-
-        self._resource_collection = assets_db.get_collection('resource')
-        self._datum_collection = assets_db.get_collection('datum')
-
-        self._collections = {'start': self._run_start_collection,
-                             'stop': self._run_stop_collection,
-                             'resource': self._resource_collection,
-                             'descriptor': self._event_descriptor_collection,
-                             'event': self._event_collection,
-                             'datum': self._datum_collection}
+        self._run_start_collection = mds_db.get_collection("run_start")
+        self._run_start_collection_revisions = mds_db.get_collection(
+            "run_start_revisions"
+        )
+        self._run_stop_collection = mds_db.get_collection("run_stop")
+        self._run_stop_collection_revisions = mds_db.get_collection(
+            "run_stop_revisions"
+        )
+        self._event_descriptor_collection = mds_db.get_collection("event_descriptor")
```

Comment:
We need `event_descriptor_revisions` too.

Reply:
Created in b86982141e4d50c272f1370baf6b132b7e269119